exercises/largest-series-product/src/example/kotlin/LSP.kt

```diff
@@ -0,0 +1,42 @@
+import java.lang.Math.max
+
+object LSP {
+
+    fun calculate(digits: String, span: Int): Long {
+        require(!digits.contains("\\D".toRegex()) )
```

Comment:
Sure, I'll fix the issues you mentioned asap!
On 10 Jun 2016 05:57, "Steve D" <notifications@github.com> wrote:

> In exercises/largest-series-product/src/example/kotlin/LSP.kt
> <https://github.com/exercism/xkotlin/pull/12#discussion_r66556163>:
>
> > @@ -0,0 +1,42 @@
> > +import java.lang.Math.max
> > +
> > +object LSP {
> > +
> > +    fun calculate(digits: String, span: Int): Long {
> > +        require(!digits.contains("\\D".toRegex()) )
>
> It may be more clear to do: require(digits.all { it.isDigit() })
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/exercism/xkotlin/pull/12/files/ab8dda84f2048845c863b9dc43949bdd3b44f356#r66556163>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AELBv1PYFbsyaY2esxPyW1d6giv335Wrks5qKNKZgaJpZM4IxxmY>
> .
>